mercury.spec

```diff
@@ -26,6 +26,7 @@ Group:    Development/Libraries
 License:  Argonne National Laboratory, Department of Energy License
 URL:      http://mercury-hpc.github.io/documentation/
 Source0:  https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
+Patch0:   https://github.com/daos-stack/mercury/na_ucx_local_addr.patch
```

Comment:
I was under the impression that recently we switched to just specifying relative path to the patch e.g.  'na_ucx_local_addr.patch' instead of full url. @brianjmurrell what is the right method now?

Reply:
As @brianjmurrell mentioned the other day, the URL like that is not valid, it needs to be specified as: `Patch0: na_ucx_local_addr.patch`

Reply:
```suggestion
Patch0:   na_ucx_local_addr.patch
```